java/src/jmri/jmrix/loconet/hexfile/ConnectionConfig.java

```diff
@@ -26,7 +26,7 @@ public ConnectionConfig() {
 
     @Override
     public String name() {
-        return "LocoNet Simulator"; // NOI18N
+        return Bundle.getMessage("TitleLocoNetSimulator");
```

Comment:
If I recall correctly, the `name()` method of the `ConnectionConfig` class returns a name which is used within the configuration profile.  Changing this line to an "internationalized" name could therefore cause difficulty for anyone with an existing configuration profile specifying this connection type who moves from an older JMRI version to a version based on this code.  That failure-mode should be avoided!  If that recollection is accurate, it justifies the use of `// NOI18N` in the original code for this method, and justifies modification to include `// NOI18N` in this method for ConnectionConfig code for other connection types.

Reply:
OK. I'm reverting to the previous version, removing the internationalization.